### ISSUE TITLE
Every page needs a level one heading. The best practice is to start the main content of a web page with a level 1 heading (<h1>), with no other headings before this high-level heading.

### ISSUE THREAD

**Issue by Paul-Clue** [LLK/scratch-www#7181]
### Where To Find Bug
The editor

### Expected Behaviour
The screen reader should announce a level one heading at the beginning of each page, according to aria best practices

### Actual Behaviour
The screen reader announces whatever heading level that was coded first in the page, potentially confusing and/or annoying AT users.

### Steps To Reproduce

- Open https://scratch.mit.edu/ in any browser on your computer
- Turn-on your screen reader of choice
- Start tabbing through the page until the screen reader announces the level of the first heading in tab focus
- Notice that there is no level one heading at the beginning of the page.

### System Details
macOs Ventura, Chrome Browser, VoiceOver (AT)

**Comment by benjiwheeler** [comment]
FWIW, the replication instructions here don’t seem to describe a gap between user experience expectation, and user experience reality. Maybe replication instructions aren’t the right approach for this type of issue?
